File: src - c++/Esercizi/EOrdinata/EOrdinata.cpp
bool Lista::EOrdinata() const {
  int i = 0;
  while (i < nelem - 1) {
    if (v[i] > v[i+1])
      return false; //esce subito se trova un'inversione
    i++;
  }
  return true; //esce senza aver trovato alcuna inversione: lista ordinata
}

File: src - java/Esercizi/EOrdinata/EOrdinata.java
	public boolean eOrdinata() {
		for (int i = 0; i < numEl - 1; i++)
			if (v[i] > v[i + 1])
				return false; // esce prematuramente se trova
				              // un'inversione

		return true; // se non ha trovato inversioni, restituisce
		             // true.
	}